CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 2.13.0-rc.0 / 2019-09-25
+
+* [CHANGE] Metrics: Change prometheus_sd_configs_failed_total to Gauge #5254
+* [CHANGE] Service discovery: add new node address types for kubernetes. #5902
```

Comment:
Is this really a CHANGE?

Reply:
ENHANCEMENT ?

Reply:
@simonpasquier could you give some context, should this be included and is it a change or an enhancment?

Reply:
updated to enhancment